front/chat/src/Xmpp/MucRoom.ts

```diff
@@ -753,6 +761,13 @@ export class MucRoom {
             } else {
               if (delay > this.lastMessageSeen) {
                 this.countMessagesToSee.update((last) => last + 1);
+                if (
+                  !get(chatVisibilityStore) ||
```

Comment:
I don't understant the `chatVisibilityStore` usage?

We play sound and send notification only if the chat is open?

Reply:
It's the opposite,we play the sound and send the notification when :
- the chat is closed 
- or if the chat is open and that the current thread is not the thread where the new message comes from